### PR TITLE
Fixes garbage subsystem

### PR DESCRIPTION
**Pull request by Carbonhell**
:cl: JohnGinnane
fix: Fixes a subsystem that was causing lag for 8 months
/:cl:

1) Fixes turfs not qdelling properly/at all due to a missing return value
2) Removes yoyo's if checks inside garbage.dm that only hide issues
3) fixes two dme entries being shuffled
special thanks to @ninjanomnom too!

### CODE REVIEW

**Comment by JohnGinnane** [comment]
Before:
image

After:
image

**Comment by ghost** [comment]
+1

File: hippiestation/code/game/turfs/turf.dm [hunk 1]
```diff
@@ -3,8 +3,6 @@
 	var/pinned = null
 
 /turf/Destroy(force)
```
Comment (JohnGinnane) [inline hippiestation/code/game/turfs/turf.dm]: remove `force`

**Review by JohnGinnane — APPROVED** [review]

**Comment by Cyberboss** [comment]
Turfs really shouldn't ever be deleted, Take a look at ours: https://github.com/tgstation/tgstation/blob/master/code/game/turfs/turf.dm#L80

**Comment by Carbonhell** [comment]
don't they get deleted in ChangeTurf, on some special cases?

**Comment by JohnGinnane** [comment]
@Cyberboss should I use another proc to piggyback onto to when a turf (walls in this case) is being deleted?

**Comment by Cyberboss** [comment]
You should always use ChangeTurf

**Pull request merged**